docs/run.md

```diff
@@ -228,11 +228,12 @@ As with the `add` command, removing a network requires that the onos-config clus
 ## Adding Applications
 
 Applications from outside of `onit` can be added to an `onit` cluster using the `onit add app` command. This command takes as
-an argument the name of a docker image to load. For example, to load the `onos-ztp` application:
+the name of the app as an argument. It also has `--image` flag that allows a user specify the image of 
+the application that wants to deploy. The user also can specify the pull policy for the image using `--image-pull-policy` flag. For example, to deploy the latest version of `onos-ztp`  application:
```

Comment:
the application that wants to deploy -> the application that should be deployed

Reply:
Fixed.